src/rulesets/tests/__snapshots__/specification.test.ts.snap

```diff
@@ -1,872 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`failing case 1: adding a camelCase when pascalCase is required 1`] = `
```

Comment:
The fact a snapshot test changed as a result of satisfying eslint changes is setting off alarm bells. We have done something to change the behavior of the rules that we may not understand.

I'd prefer to preserve existing behavior and make eslint exceptions until we have actual unit tests (not just snapshot coverage) to feel confident in making these changes.

Reply:
the snapshot has not changed, it was abandoned. We no longer have `specification.test.ts` file and a snapshot. That's why.